tests/lib/ignored-paths.js

```diff
@@ -164,6 +164,30 @@ describe("IgnoredPaths", () => {
                 )
             );
         });
+
+
+    });
+
+    describe("caching file reads", () => {
+
+        let readFileSyncCount;
+
+        before(() => {
+            readFileSyncCount = sinon.spy(fs, "readFileSync");
```

Comment:
Yes, that's the one. If one isn't being used elsewhere, I won't insist that you use one in this PR- we can always incorporate sandboxes later in a chore PR. Thanks for confirming!